src/tests/java/htsjdk/samtools/util/SamLocusIteratorTest.java

```diff
@@ -93,6 +121,43 @@ public void testEmitUncoveredLoci() {
                 expectedReads = 0;
             }
             Assert.assertEquals(li.getRecordAndPositions().size(), expectedReads);
+            // make sure that we are not accumulating indels
+            Assert.assertEquals(li.getDeletedInRecord().size(), 0);
+            Assert.assertEquals(li.getInsertionInRecord().size(), 0);
+        }
+        Assert.assertEquals(pos, 100001);
+
+    }
+
+    @Test
+    public void testEmitUncoveredLociWithIndels() {
```

Comment:
Sure, it will also reduce the number of methods (I will do it also with `testBasicIterator` to avoid the suggested `testIteratorWithIndelsOnButNoIndels`) in the previous comment.